app/assets/javascripts/views/challenge_modal/challenge_modal.ts

```diff
@@ -118,34 +132,19 @@ class ChallengeModalCtrl extends PureViewCtrl {
   }
 
   reloadProcessingStatus() {
-    this.setState({
-      processing: this.processingTypes.length > 0
+    return this.setState({
+      processing: this.state.processingPrompts.length > 0
     });
   }
 
-  get title(): string {
+  get modalTitle(): string {
     if (this.challenge.reason === ChallengeReason.Migration) {
       return STRING_STORAGE_UPDATE;
     } else {
       return STRING_AUTHENTICATION_REQUIRED;
     }
   }
```

Comment:
Is the title not stored in the challenge now? If so we shouldn't use our own strings here

Reply:
I believe this is the modal window title, which SNJS doesn't handle. Within the modal we have a prompt and subprompt, which is the title and subtitle in SNJS.